llvm-obs/obs-scripts/dod-ubuntu-init.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Generates dod repos for all supported Ubuntu versions in obs
```

Comment:
A bunch of them have been deprecated already

Reply:
But snapshot branch has build job for all these in the Jenkins instance? 
And for branch 6.0 we don't build for all these. Are you referring to that? :)